src/components/inputdate/InputDate.js

```diff
@@ -3,14 +3,14 @@ import { Calendar } from '~/assets/image';
 
 const InputDate = ({ label, placeholder, modifier }) => {
   return (
-    <div className={modifier ? "input--short" : "input" }>
+    <div className={modifier ? 'input--short' : 'input'}>
       <label className="input__label">{label}</label>
-      <div className={"input__date"}>
+      <div className={'input__date'}>
```

Comment:
can't it just be `className='input__date'`?

Reply:
Actually, it's the component pulled from dev branch itself, I will update that.